accounts/abi/bind/backends/simulated.go

```diff
@@ -590,7 +590,7 @@ func (b *SimulatedBackend) callContract(ctx context.Context, call ethereum.CallM
 		return nil, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
 	}
 	head := b.blockchain.CurrentHeader()
-	if !b.blockchain.Config().IsLondon(head.Number) {
+	if !b.blockchain.Config().IsBanach(head.Number) {
```

Comment:
Behavior after London and before Banach, assuming users specify `gasPrice`:

Original behavior: Step into else branch, then if branch, `call.GasFeeCap, call.GasTipCap = call.GasPrice, call.GasPrice`.
New Behavior: Step into if branch, `call.GasFeeCap, call.GasTipCap = call.GasPrice, call.GasPrice`

Identical